js/researches/getPassiveVoice.js

```diff
@@ -37,12 +54,14 @@ var matchArray = function( sentence, matches ) {
 
 	forEach( matches, function( part ) {
 		part = stripSpaces( part );
+		var indices = getIndicesOf( part, sentence );
```

Comment:
This line is being executed early on, although it's possible that there are no matches. I'd place this line _after_ `if ( !matchWordInSentence( part, sentence ) ) { ... }`.